amethyst_ui/src/prefab.rs

```diff
@@ -414,6 +439,33 @@ pub struct UiButtonBuilder<W: WidgetId = u32> {
     pub release_sound: Option<AssetPrefab<Audio>>,
 }
 
+impl<W: WidgetId + Debug> Debug for UiButtonBuilder<W> {
```

Comment:
This could be implemented using `derivative` with custom format just `for` font prop
https://mcarton.github.io/rust-derivative/Debug.html#format-with

Reply:
format_with is currently broken with latest rust and Derivative.